bin/s2nc.c

```diff
@@ -238,6 +245,31 @@ int main(int argc, char *const *argv)
 
     struct s2n_connection *conn = s2n_connection_new(S2N_CLIENT);
 
+    if (mfl_value > 0) {
+        switch(mfl_value) {
+            case 512:
+                mfl_code = S2N_TLS_MAX_FRAG_LEN_512;
+                break;
+            case 1024:
+                mfl_code = S2N_TLS_MAX_FRAG_LEN_1024;
```

Comment:
Makes sense. I initially thought about requiring enum values instead of the actual mfl values, but thought it may be more straightforward for users to input a mfl value.

AS you mentioned, since the values are exposed. It is ok for users and also simplifies our logic here.